output-comparator/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     implementation 'com.beust:jcommander:1.48'
     implementation 'com.google.code.gson:gson:2.8.6'
     implementation 'com.github.stefanbirkner:system-lambda:1.2.0'
```

Comment:
Can dependency on `'com.github.stefanbirkner:system-lambda:1.2.0'` be removed now?

Reply:
Oh right, I didn't notice. Good catch.